### DISCUSSION TITLE
Spatial output analysis for SOC

### DISCUSSION THREAD

**Discussion by KeLiu7** [magpiemodel/magpie#438] [Help]
Hi Jan,

I already did a default running.

![image](https://user-images.githubusercontent.com/72141690/186606881-499e25ec-abde-4c8e-b6a3-d33489553188.png)

When I open the nc file with Panol or mz file with R, those results are mainly for crops... 

Could you please tell me where I can find output results relevant with SOC, livestock and GHG emissions? 

Looking forward to your suggestions! Thanks in advance.

**Comment by tscheypidi** [comment]
not sure if we currently have them already on grid level. You can have a look at the various output scripts we have and check whether there is perhaps one available providing you with the information you are looking for (running `Rscript output.R` will give you an overview with brief descriptions of each script). If not available already, you probably have to write your own output script for that purpose.

**Reply by KeLiu7** [reply]
Thanks Jan. Already solved the disaggregation of SOC

**Reply by flohump** [reply]
How did you do the disaggregation of SOC?

**Reply by KeLiu7** [reply]
> How did you do the disaggregation of SOC?

You can use SOM(gdx = "fulldata.gdx", level = "grid"). Before running this, you might need load `getDirectory` function.

Hope this works for you.

**Reply by KeLiu7** [reply]
Hi Jan and Kristine,  @tscheypidi @k4rst3ns 

After I run the codes `SOM(gdx = "fulldata.gdx", level = "grid")`, it says some warnings below. Are these important? 

Although this is a run with cc: half-degree soil carbon pattern of 1995 is used for disaggregation of carbon stocks from cluster to cell level. Note that for that reason no changes in soil carbon stock patterns due to cc below cluster level are present.Warning message:
In SOM(gdx = "fulldata.gdx", level = "grid") :
  Disaggregation on grid level will not conserve total carbon stocks, but cshares.
This leeds to a mismatch in cropland carbon stock of over 10% in: MEA. On cluster level this is even worse.
